index.html

```diff
@@ -10439,23 +10491,34 @@ <h2 id="WaveShaperNode">
               sequence&lt;float&gt; real
             </dt>
             <dd>
-              The array of <code>cosine</code> terms equivalent to the
-              <a><code>real</code></a> parameter to <a href=
-              "#widl-BaseAudioContext-createPeriodicWave-PeriodicWave-Float32Array-real-Float32Array-imag-PeriodicWaveConstraints-constraints">
-              <code>createPeriodicWave</code></a>. This defaults to a sequence
-              of all zeroes of the same length as <a><code>imag</code></a> if
-              <a><code>imag</code></a> is given.
+              <p>
+                The <dfn id="dfn-real">real</dfn> parameter represents an array
+                of <code>cosine</code> terms (traditionally the A terms). In
+                audio terminology, the first element (index 0) is the DC-offset
+                of the periodic waveform. The second element (index 1)
+                represents the fundamental frequency. The third element
+                represents the first overtone, and so on. The first element is
+                ignored and implementations must set it to zero internally.
+              </p>
+              <p>
+                This defaults to a sequence of all zeroes of the same length as
+                <a><code>imag</code></a> if <a><code>imag</code></a> is given.
+              </p>
             </dd>
             <dt>
               sequence&lt;float&gt; imag
             </dt>
             <dd>
-              The array of <code>sine</code> terms terms equivalent to the
-              <a><code>imag</code></a> parameter to <a href=
-              "#widl-BaseAudioContext-createPeriodicWave-PeriodicWave-Float32Array-real-Float32Array-imag-PeriodicWaveConstraints-constraints">
-              <code>createPeriodicWave</code></a>. This defaults to a sequence
-              of all zeroes of the same length as <a><code>real</code></a> if
-              <a><code>real</code></a> is given.
+              <p>
+                The <dfn id="dfn-imag">imag</dfn> parameter represents an array
+                of <code>sine</code> terms (traditionally the B terms). The
+                first element (index 0) should be set to zero (and will be
```

Comment:
I think you should make "and will be ignored" part written consistently for both real and imag elements. So make the both parenthetical as here or both complete sentences as done above for the real element.